src/modules/integrator-guide/pages/compose-configuration/duplication-detection.adoc

```diff
@@ -5,10 +5,122 @@ include::ROOT:partial$variables.adoc[]
 {PRODUCT_NAME} through the duplicate detection feature helps maintain clean and accurate data by preventing record duplication.
 This is achieved by using rules to apply a criterion that determines how closely a field on a new or edited record matches the same field on an existing record.
 
-The user interface displays a warning or an error message to alert the user of an existing record with the exact field values within a module thereby preventing duplicates.
+The user interface displays a warning or an error message to alert the user of an existing record with the exact field values within a module, thereby preventing duplicates.
 
 .Benefits of corteza duplicate detection:
 * Maintain data quality,
 * improves accuracy and efficiency in data,
 * enhances effective description of records,
 * optimizes data storage.
+
+== Validation Rules
+
+The duplicate detection feature is enabled by using the value validation rules on a module.
+Validation rules verify that the designated fields in a record do not contain the same values, therefore ensuring that no duplicate values are saved when creating or editing a record.
+
+Validation can be configured using two rules:
+
+* Strict duplicate value validation
+
+This rule ensures that a record will not be saved when a duplicate value is detected in the selected fields in any existing record of this module. 
+
+When the Strict duplicate value validation rule is applied, an error message pop-up will be displayed on the bottom right side of the screen as an indication that a record with the same values exists.
+
+* Soft duplicate value validation
+
+This rule allows a record to be saved but issues a warning message pop-up when a duplicate value is detected, indicating that there is an existing record with the exact field values within the module.
+
+When Soft duplicate value validation is applied, the warning message is displayed on the bottom right side of the screen.
+
+=== Enabling value validation rules:
+
+[NOTE]
+====
+Validation rules can be enabled when creating new modules or on exiting modules.
+====
```

Comment:
I don't think this needs to be in an admonition since it's not really a big deal, and it's already implied by the following line

> Navigate to your Low Code namespace. Click on an existing module or create a new module.

Reply:
okay